CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 ### ⬆️ Improved
 - Offline data is clear after the user is disconnect by calling `ChatClient.disconnect(true)`. [#3917](https://github.com/GetStream/stream-chat-android/pull/3917)
 - Adding logs to understand more about unrecoverable errors in socket connection. [#3946](https://github.com/GetStream/stream-chat-android/pull/3946)
+- Refactor in the initialization state. Now you can check when the current state of the intialization progress and check if an error happened. [#3962](https://github.com/GetStream/stream-chat-android/pull/3962)
```

Comment:
Can we add more info here? Let's mention which property was added/changed

Reply:
This text need to be updated, There is not "an error" state now